robot_skills/test/test_speech_recognition.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 import sys
-
 import rospy
+from robot_skills import get_robot
 
 if len(sys.argv) < 2:
```

Comment:
Add argument check like other files to be consistent

Reply:
See a71172c306547eba6d50c057baaef8c0ec10a1d0 and 106535870505f785620744139c45820da4f330a4